### DISCUSSION TITLE
Patreon alternative

### DISCUSSION THREAD

**Discussion by vbooka1** [mriscoc/Sparrow-Extended-UI-for-RIGOL-DHO800_DHO900#6] [General]
Hello, my religion forbids using Patreon, is there any other way to purchase the mod?

**Comment by mriscoc** [comment]
You can contribute using  [Paypal](https://www.paypal.com/donate/?business=85SPAAR6UZEE8)   

[<img src="https://www.paypalobjects.com/en_US/i/btn/btn_donateCC_LG.gif">](https://www.paypal.com/donate?business=85SPAAR6UZEE8&currency_code=USD) 

Please make sure your email address in Paypal is correct. I will send an email to the address on file to initiate contact.

**Reply by vbooka1** [reply]
Unfortunately Paypal is not an option too. Could I send you a SEPA transfer? I have an account in (Transfer)Wise.

![ffdf](https://github.com/mriscoc/RIGOL_DHO800_DHO900_GUI/assets/29492874/a2fbab90-ee94-457e-9ee1-160d64263c64)

**Comment by mriscoc** [comment]
Try with this: https://github.com/sponsors/mriscoc

**Reply by vbooka1** [reply]
Done, 

>  🎉 You sponsored @mriscoc for $15 one time

**Reply by mriscoc** [reply]
Thank you, please give me your email
